src/Field.php

```diff
@@ -223,6 +223,21 @@ public function __construct(array $defaults = [])
         }
     }
 
+    protected function onHookToOwner(string $spot, \Closure $fx, array $args = [], int $priority = 5): int
+    {
+        $name = $this->short_name;
+
+        return $this->owner->onHookDynamic(
+            $spot,
+            static function (Model $owner) use ($name) {
+                return $owner->getField($name);
```

Comment:
Why not `return $owner->getField($this->short_name);`

Reply:
because we will prevent old `$this` to GC, goiing to sleep today